civihr_employee_portal/js/reports.js

```diff
@@ -18,6 +19,43 @@
   HRReport.prototype.init = function (options) {
     $.extend(this, options);
     this.processData(this.data);
+    this.originalFilterElement = jQuery('#report-filters').detach();
+  };
+
+  /**
+   * Compiles the filters element into an angular component. This is done for
+   * elements created outside of the Angular cycle (ie: created using jQuery, etc.)
+   */
+  HRReport.prototype.appendFilters = function () {
+    var element = this.originalFilterElement.clone();
```

Comment:
The original element is kept in memory (removed from the DOM), but the clone is the one that actually gets attached to the DOM since the clone can be removed at any time when the report is updated.